src/components/quiz/QuizQuestions.tsx

```diff
@@ -1,18 +1,91 @@
-import React from "react";
+import React, { useState } from "react";
 import { Question } from "./questions";
-import { Reward } from "./Reward";
+import { UserDetailsForm } from "./UserDetailsForm";
+import './QuizQuestions.scss';
 
 interface QuizQuestionsProps {
-  questions: Question[];
+   questions: Question[];
+   score: number;
+   setScore: (score: number) => void;
+   isEndOfQuestion: boolean;
+   setIsEndOfQuestion: (isEndOfQuestion: boolean) => void;
 }
 
+
 export const QuizQuestions: React.FunctionComponent<QuizQuestionsProps> = ({
-  questions,
+  questions,score,setScore,isEndOfQuestion,setIsEndOfQuestion
```

Comment:
Put these all on their own lines
```suggestion
  questions,
  score,
  setScore,
  isEndOfQuestion,
  setIsEndOfQuestion,
```